### ISSUE TITLE
About the COCO DataSet

### ISSUE THREAD

**Issue by fangpengcheng95** [facebookresearch/Detectron#59]
1 : 
I download the COCO(2014) dataset from cocodataset.org website,that has the following file structure:
_ train2014
|  |_ <im-1-name>.jpg
|  |_ ...
|  |_ <im-N-name>.jpg
|_ val2014
|_ test2014
|_ annotations
   |_ instances_train2014.json
   |_ instances_val2014.json
   |_ captions_train2014.json
   |_ captions_val2014.json
   |_ person_keypoints_train2014.json
   |_ person_keypoints_val2014.json
   |_ image_info_test2014.json

compare with the coco dataset that define in your Detectron/lib/datasets/data/README.md 
coco
|_ coco_train2014
|  |_ <im-1-name>.jpg
|  |_ ...
|  |_ <im-N-name>.jpg
|_ coco_val2014
|_ ...
|_ annotations
   |_ instances_train2014.json
   |_ ...

should i just change the name of  the directory from  "train2014" to "coco_train2014" && "val2014" to "coco_val2014" && "test2014" to "coco_test2014", does this work?

2:
where can i get the dataset COCO Minival?

**Comment by fangpengcheng95** [comment]
"minval" == "2017 val"
"valminusminival" + "2014 train" == "2017 train"

**Comment by ZhuoyaYang** [comment]
@fangpengcheng95  Can you explain what is valminusminival？I do not understand.The fine-tune model labeled "coco_2014_train+coco_2014_valminusminival" .

**Comment by muzammil360** [comment]
@fangpengcheng95, images in 2017 validation set don't have the same naming convention as 2014. So simply renaming the folders doesn't work. 

Are you saying that we need to rename each and every file in 2017 validation set to make 2014 mini val? 

And how do we make the set valminusminival without comparing every single image of 2017 train with 2014 train?

My whole point is: Is there any place where we can just download these folders in the right format or does every one does the hardwork of copy and renaming themselves?

**Comment by muzammil360** [comment]
After much work, I found the solution to the question I asked above. All you have to do is download the annotations from [here](https://github.com/rbgirshick/py-faster-rcnn/blob/master/data/README.md). We don't need to split val2014 into two separate folders. The data remains in val2014 folder.

**Comment by nitheeshkl** [comment]
update: details for COCO minival annotations are provided here - https://github.com/facebookresearch/Detectron/blob/master/detectron/datasets/data/README.md
